content/docs/user-guide/experiment-management/cleaning.md

```diff
@@ -0,0 +1,262 @@
+# Cleaning Up Experiments
+
+Although DVC uses minimal resources to keep track of the experiments, they may
+clutter tables and the workspace. DVC allows to remove particular experiments
+from the workspace or delete all not-yet-persisted experiments at once.
+
+## Removing Local Experiments
+
+When you want to discard an experiment you can use `dvc exp remove` and supply
+the experiment name.
+
+```dvc
+$ dvc exp list
+main:
+    cnn-32
+    cnn-64
+    cnn-128
+$ dvc exp remove cnn-32
+$ dvc exp list
+main:
+    cnn-64
+    cnn-128
+```
+
+## Removing Multiple Local Experiments
```

Comment:
Here I'd go for "unnecessary" (implies plural) which is the intention of `gc`.

```suggestion
## Removing unnecessary experiments
```

Reply:
Necessary/unnecessary seems like something we can't decide 😄 

I'll go for "Removing All Experiments"

Reply:
> Necessary/unnecessary seems like something we can't decide

The user will decide but that's what `exp gc` is for so let's be consistent with our language?

![image](https://user-images.githubusercontent.com/1477535/131704172-16ccd910-d2d0-40ad-a88e-21d4faf3c7d1.png)